pkg/trace/stats/concentrator_test.go

```diff
@@ -210,7 +214,7 @@ func TestTracerHostname(t *testing.T) {
 	traceutil.ComputeTopLevel(spans)
 	testTrace := toProcessedTrace(spans, "none", "tracer-hostname", "", "", "")
 	c := NewTestConcentrator(now)
-	c.addNow(testTrace, "")
+	c.addNow(testTrace, "", nil)
```

Comment:
We should add a test for having an actual slice of container tags. I know we have other tests that cover the end-to-end integration (so to speak), but I think it would make sense to have a case (or more) here too.

Reply:
SG, added to concentrator_test